src/NanoProtocol.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <nanomsg/nn.h>
+#include <string>
+/*
+ * class for encapsulating nanomsg zero copy allocation/deletion
+ */
+class NanoProtocol {
+public:
+   enum class Protocol {TCP,IPC};
```

Comment:
@KjellKod same as before, do we realy want to do this? as it will come with a performance penalty. i have no preference either way.